x/model/client/cli/tx_model.go

```diff
@@ -102,6 +106,8 @@ func CmdCreateModel() *cobra.Command {
 		"Model description (string or path to file containing data)")
 	cmd.Flags().StringVar(&partNumber, FlagPartNumber, "",
 		"Model Part Number (or sku)")
+	cmd.Flags().Uint32Var(&discoveryCapabilitiesBitmask, FlagDiscoveryCapabilitiesBitmask, 0,
```

Comment:
Let's add a validation to CLI that if FlagCommissioningFallbackURL is specified, then FlagDiscoveryCapabilitiesBitmask, must also be specified.

Reply:
Added validation